src/FFmpegReader.cpp

```diff
@@ -1766,8 +1788,8 @@ void FFmpegReader::Seek(int64_t requested_frame) {
 		bool seek_worked = false;
 		int64_t seek_target = 0;
 
-		// Seek video stream (if any)
-		if (!seek_worked && info.has_video) {
+		// Seek video stream (if any), except album arts
+		if (!seek_worked && info.has_video && !HasAlbumArt()) {
```

Comment:
I'm concerned the same issue may bite us here: MP4 and MKV video files _can_ have attached picture data.